### DISCUSSION TITLE
Category carousel problem

### DISCUSSION THREAD

**Discussion by sudeep2003** [bagisto/bagisto#10172] [Q&A] [answered]
I’m facing an issue with the category carousel. Currently, it automatically displays categories connected to the `root` level, which is problematic for my use case. 

For example, I have a parent category called "Man," and a subcategory named "Special for men."

I want to display "Special for men" in the carousel, but there is no option to show it. Additionally, I want to exclude the parent category "Man," but there is no option to hide it either.

Is there a way to customize the category carousel to display specific subcategories without showing their parent categories? If so, could you please provide guidance on how to achieve this?

Thank you!
![Screenshot (546)](https://github.com/user-attachments/assets/eca82dc4-420f-4ff8-bc80-4cbdc19e7f75)

**Comment by amit-webkul** [comment, marked as answer]
Hello @sudeep2003 

Currently, in the flow, we are displaying only the categories whose parent is the root. For this, we are using the API, with the file path mentioned below:

https://github.com/bagisto/bagisto/blob/9b9a32a0aedf1b8d56f587e36653c835f8163c22/packages/Webkul/Shop/src/Http/Controllers/API/CategoryController.php#L50

If you want to make changes or updates, you can add your modifications inside the API function.
